app/views/users/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>Here is individual user </h1>
\ No newline at end of file
+      <h1 class="user-name"><%=@user.name%><span class="posts_counter">Number of Posts: <%=@user.posts_counter%></span></h1>
+<h3 class="bio">Bio: <p ><%=@user.bio%></p></h3>
+<ul>
+<%@user.recent_posts.each_with_index do |post, index|%>
+  
+<li >Post #<%= index + 1%>
+<h6><%=post.title%></h6> 
+
+<div class="comment-likes">
+<span>Comments: <%=post.comments_counter%></span>
+<span class="like-span">Likes: <%=post.likes_counter%></span>
+</div>
+
+</li>
+
+<%end%>
```

Comment:
- [ ] @Lul-Abdifan Great work adding the user show page, however, it's missing a link at the end of the page `see all posts` that takes the users to the posts index page 
![image](https://github.com/Lul-Abdifan/Blog-App/assets/61408860/fb7cfda6-d5e4-46e1-9526-f564da55ed23)